ci/jenkins/jobs/macros.yaml

```diff
@@ -252,7 +252,29 @@
           return_code=$?
           set -ex
           ./ci/kind/kind-setup.sh destroy "${{FULL_CLUSTER_NAME}}"
-          exit $return_code 
+          exit $return_code
+                   
+- builder:
+    name: builder-kind-flexible-ipam-e2e
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          KIND_TIMEOUT=135
+          IPAM_VLAN_SUBNETS='192.168.241.1/24,192.168.242.1/24'
+          IPAM_VLAN_IDS='11,12'
```

Comment:
I would say this belongs to test scripts but I understand why they must be specified here.

In the future, we should move kind tests to "test-e2e-kind.sh", instead of using test.sh for mixed purposes.